src/ts/Signin/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { Link } from 'react-router'
+import Container from '../Container'
+import SigninForm from './SigninForm'
+
+export default function Signin(props) {
+  return (
+    <Container className="cont">
+      <div className="form">
+        <h1 className="title">Create a new account</h1>
+        <div className="title-sub">
+          Already have an account?&#160;
```

Comment:
space char, I changed it to no-breaking space `&nbsp;` http://andrewhfarmer.com/how-whitespace-works-in-jsx/